### DISCUSSION TITLE
Aura tries to downgrade to a signature of a package

### DISCUSSION THREAD

**Discussion by etdr** [fosskers/aura#763] [Q&A] [answered]
I'm so sorry because I am sure this has happened to other people but is there a reason why when I go to downgrade recent packages Aura identifies the .zst.sig file, not the .zst itself? Did I turn on some setting that separates out the signature or something?

**Comment by fosskers** [comment, marked as answer]
Let me look into this.

**Reply by fosskers** [reply]
```
aura >>= What version of xterm do you want?
1. /var/cache/pacman/pkg/xterm-370-2-x86_64.pkg.tar.zst
2. /var/cache/pacman/pkg/xterm-371-1-x86_64.pkg.tar.zst
3. /var/cache/pacman/pkg/xterm-372-1-x86_64.pkg.tar.zst.sig
4. /var/cache/pacman/pkg/xterm-372-2-x86_64.pkg.tar.zst.sig
```
Confirmed, it should not be doing that! Note that this bug doesn't exist in the Rust branch, but I'll fix the Haskell anyway.

**Reply by fosskers** [reply]
I've released `3.2.9` which fixes this bug. Thanks for reporting it.

**Reply by etdr** [reply]
No problem! Should I go ahead and swich to Rust at this point?

**Reply by fosskers** [reply]
No the Rust isn't quite ready yet, but it's getting there.
